CDCgov/NEDSS-Modernization
# JDBC_DRIVER -> Dockerfile

## Description

When removing the environment variables from the NiFi Dockerfile, the `JDBC_DRIVER` variable was erroneously removed. This PR adds this back to the Dockerfile

## Tickets

NA

## Steps to verify

1. Rebuild NiFi: In the `cdc-sandbox` directory run `docker-compose up nifi --build -d`
1. After container startup navigate to `https://localhost:8443/nifi/`
2. Login with default credentials
3. Verify there are no errors and the NiFi flow was able to successfully ingest data
